### PR TITLE
Add all, network, permissions modifiers

### PR DESCRIPTION
**Pull request by jellizaveta**
The network modifier only corresponds to IP addresses and supports regexp. I made a regexp against these two criteria, but regexp should also point to an ip address, should this be taken into some way?

### CODE REVIEW

File: src/compatibility-tables/modifiers/network.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+adg_os_any:
+  name: network
+  description: This is basically a Firewall-kind of rules allowing to fully block or unblock access to a specified remote address.
+  docs: https://adguard.com/kb/general/ad-filtering/create-own-filters/#network-modifier
+  conflicts:
+    - app
+  inverse_conflicts: true
+  assignable: true
+  # value_format: /^(?:\d{1,3}\.){3}\d{1,3}(?::\d+)?$|^\[?[0-9a-fA-F:]+\]?(?::\d+)?$|^\/(?:[^\/\\]|\\.)*\/[a-z]*$/
```
Comment (scripthunter7) [inline src/compatibility-tables/modifiers/network.yml:9]:
```suggestion
  # TODO: Implement validation
  # value_format: /^(?:\d{1,3}\.){3}\d{1,3}(?::\d+)?$|^\[?[0-9a-fA-F:]+\]?(?::\d+)?$|^\/(?:[^\/\\]|\\.)*\/[a-z]*$/
```

**Review by scripthunter7 — CHANGES_REQUESTED** [review]
Please add TODO comments, see my suggestions

File: src/compatibility-tables/modifiers/permissions.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+adg_os_any:
+  name: permissions
+  description: For the requests matching a $permissions rule, ad blocker strengthens response's feature policy by adding additional feature policy equal to the $permissions modifier contents. $permissions rules are applied independently from any other rule type.
+  docs: https://adguard.com/kb/general/ad-filtering/create-own-filters/#network-modifier
+  version_added: "CoreLibs 1.11"
+  assignable: true
+  # value_format:
```
[the remark this answers is not in the document]
Comment (scripthunter7) [reply src/compatibility-tables/modifiers/permissions.yml:7]: @jellizaveta `value_format` always refers to the value assigned to the modifier in the following form: `$modifier=assigned_value`

There is currently no planned field in the compatibility table for checking the pattern before the modifier list

[the remark this answers is not in the document]
Comment (scripthunter7) [reply src/compatibility-tables/modifiers/permissions.yml:7]: cc @slavaleleka

File: src/compatibility-tables/modifiers/network.yml (hunk 1 above)
Comment (scripthunter7) [reply src/compatibility-tables/modifiers/network.yml:9]: @jellizaveta `value_format` always refers to the value assigned to the modifier in the following form: `$modifier=assigned_value`

There is currently no planned field in the compatibility table for checking the pattern before the modifier list

CC @slavaleleka

File: src/compatibility-tables/modifiers/permissions.yml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+adg_os_any:
+  name: permissions
+  description: For the requests matching a $permissions rule, ad blocker strengthens response's feature policy by adding additional feature policy equal to the $permissions modifier contents. $permissions rules are applied independently from any other rule type.
+  docs: https://adguard.com/kb/general/ad-filtering/create-own-filters/#network-modifier
```
Comment (scripthunter7) [inline src/compatibility-tables/modifiers/permissions.yml:4]:
```suggestion
  docs: https://adguard.com/kb/general/ad-filtering/create-own-filters/#permissions-modifier
```

**Review by scripthunter7 — CHANGES_REQUESTED** [review]

**Review by Alex-302 — APPROVED** [review]

**Review by zloyden — APPROVED** [review]

**Review by scripthunter7 — APPROVED** [review]
LGTM, but please wait for @slavaleleka

**Review by slavaleleka — APPROVED** [review]

**Pull request merged**